src/chat/chatReducers.js

```diff
@@ -70,6 +72,29 @@ const eventMessageDelete = (
   return stateChange ? newState : state;
 };
 
+const eventUpdateMessageFlags = (
+  state: NarrowsState,
+  action: EventUpdateMessageFlagsAction,
+): NarrowsState => {
+  const { messages, flag, operation } = action;
+  const sources = [state];
+
+  if (state[STARRED_NARROW_STR] && flag === 'starred') {
```

Comment:
Nit: I'd prefer `flag === 'starred'` first. That gives this the shape of an overall `switch/case` on `flag`, which is the appropriate shape for this to have in general.  The `state[STARRED_NARROW_STR]` is logic specific to the case when we're talking about the `starred` flag at all.

Reply:
Updated.